test/testWorkerBee.js

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (C) 2016  Max Prettyjohns
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+'use strict';
+
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+chai.use(chaiAsPromised);
+const expect = chai.expect;
+const rewire = require('rewire');
+const Promise = require('bluebird');
+const testData = require('../data/testData.js');
+const Achievement = rewire('../src/server/helpers/achievement.js');
+
+const workerBeeIThreshold = 1;
+const workerBeeIIThreshold = 10;
+const workerBeeIIIThreshold = 100;
+
+
+module.exports = function tests() {
+	beforeEach(() => testData.createWorkerBee());
+
+	afterEach(testData.truncate);
+
+	it('I should be given to someone with a Work creation',
```

Comment:
Probably best to have a lowercase "work" here to be consistent with the other descriptions